pkg/log/context.go

```diff
@@ -23,46 +23,66 @@ import (
 	"os"
 	"sync"
 
+	"github.com/onsi/ginkgo"
+
 	"github.com/onrik/logrus/filename"
+
 	"github.com/sirupsen/logrus"
 )
 
 type logKey struct{}
 
+const (
+	// FieldComponentName is the key of the component field in the log message.
+	FieldComponentName = "component"
+	// FieldCorrelationID is the key of the correlation id field in the log message.
+	FieldCorrelationID = "correlation_id"
+)
+
 var (
+	defaultEntry = logrus.NewEntry(logrus.StandardLogger())
+
 	supportedFormatters = map[string]logrus.Formatter{
 		"json": &logrus.JSONFormatter{},
 		"text": &logrus.TextFormatter{},
 	}
-	regMutex     = sync.Mutex{}
-	once         = sync.Once{}
-	defaultEntry = logrus.NewEntry(logrus.StandardLogger())
+
+	supportedOutputs = map[string]io.Writer{
```

Comment:
No way to add other outputs. What if we want to log to a file?

Reply:
I could add logic for this  but whats the point of we cant change the log output from the API (your other comments)